### PR TITLE
add php+redis k8s deployment

### PR DESCRIPTION
**Pull request by sophia0328**

### CODE REVIEW

File: php-fpm/README.md [hunk 1]
```diff
@@ -56,8 +56,42 @@ image](https://hub.docker.com/_/php).
...
 ### Deploy with Kubernetes
-
-
+This image can also be deployed on a Kubernetes cluster, such as
+[minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).The
+following example YAML files are provided in the repository as
+reference for Kubernetes deployment:
+
+   * [`pv-local.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/php-fpm/pv-local.yaml), [`pvc-local.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/php-fpm/pvc-local.yaml):
+     local persistent volumes for databse and wordpress.
```
Comment (ZhongbaoShi) [inline php-fpm/README.md]: is this related with Wordpress?

File: php-fpm/README.md [hunk 2]
```diff
@@ -56,8 +56,42 @@ image](https://hub.docker.com/_/php).
...
+   * [`redis-deployment.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/php-fpm/redis-deployment.yaml):
+     deployment for redis.
+
+The example utilies nginx, php-fpm and redis containers. All are Clear Linux
+published ones on [official clearlinux base
+image](https://hub.docker.com/_/clearlinux).
+To deploy the image on a Kubernetes cluster you have two volumes configuration
+choices:
+
+   * Use local volume, not appropriate for scaling on multi-node cluster.
```
Comment (ZhongbaoShi) [inline php-fpm/README.md]: maybe modify words to support signal node only. Then, we can plan to support multi-node scale as 2nd priority

File: php-fpm/README.md (hunk 1 above)
Comment (sophia0328) [reply php-fpm/README.md]: updated. please review the new submit https://github.com/clearlinux/dockerfiles/pull/256
